src/main.rs

```diff
@@ -265,15 +265,27 @@ async fn handle_get_digest(tag: String) -> Response {
 async fn main() {
     let args: Vec<String> = env::args().collect();
 
+    if args.len() < 2 {
+        println!("Usage: {} <sha256sum> [ipv6_bind]", args[0]);
```

Comment:
Does this `true` flag offer downstream preferences with regards to complexity to manage versus standard conventions for flags such as `--ipv6` or even a more configurable `--bind=[::]:5000` option? Not a blocker.

Reply:
Technically it does make the downstream simpler as you can just pass a boolean through Helm values, but I wasn't considering that when I did this. It's easy enough to pass in any flag based on a boolean using Helm templating. 

Probably it is best to provide and most clear to provide a configurable bind here